chebi-update/src/main/java/org/reactome/release/chebiupdate/ChebiUpdater.java

```diff
@@ -115,76 +152,112 @@ public void updateChebiReferenceMolecules() throws SQLException, Exception
 			String chebiName = entity.getChebiAsciiName();
 			List<DataItem> chebiFormulae = entity.getFormulae();
 
-			updateReferenceEntities(molecule, chebiName, instanceEdit);
 
-			// Now, check to see if we need to update the ReferenceMolecule itself.
 			String moleculeIdentifier = (String) molecule.getAttributeValue(ReactomeJavaConstants.identifier);
 			String moleculeName = (String) molecule.getAttributeValuesList(ReactomeJavaConstants.name).get(0);
 			String moleculeFormulae = (String) molecule.getAttributeValue(ReactomeJavaConstants.formula);
 
 			String prefix = "ReferenceMolecule (DB ID: " + molecule.getDBID() + " / ChEBI ID: " + moleculeIdentifier + ") has changes: ";
 
```

Comment:
Prefix could also become a simple private method using molecule as a parameter.  Then prefix could be removed as a parameter from other methods and just called for inside them.